src/main/java/com/adobe/commerce/cif/api/CartApi.java

```diff
@@ -345,6 +347,40 @@ Cart deletePayment(
         @ApiParam(value = ACCEPT_LANGUAGE_DESC)
         @HeaderParam(ACCEPT_LANGUAGE) String acceptLanguage);
 
+    @DELETE
+    @Path("/{id}/payment/{paymentId}")
```

Comment:
Should be `payments` with `s`. I think the POST operation should also be changed to use plural, so you should deprecate the previous method and add a new one.

Reply:
@cjelger done.